synapse/models/infotech.py

```diff
@@ -358,6 +397,129 @@ def getModelDefs(self):
+                    }),
+                    ('domain', ('it:domain', {}), {
+                        'doc': 'The authentication domain where the account is registered.',
+                    }),
+                    ('posix:uid', ('int', {}), {
+                        'doc': 'The user ID of the account.',
+                        'ex': '1001',
+                    }),
+                    ('posix:gid', ('int', {}), {
+                        'doc': 'The primary group ID of the account.',
+                        'ex': '1001',
+                    }),
+                    ('posix:gecos', ('int', {}), {
+                        'doc': 'The GECOS field for the POSIX account.',
+                    }),
+                    ('posix:home', ('file:path', {}), {
+                        'doc': "The path to the POSIX account's home directory.",
+                        'ex': '/home/visi',
+                    }),
+                    ('posix:shell', ('file:path', {}), {
+                        'doc': "The path to the POSIX account's default shell.",
+                        'ex': '/bin/bash',
+                    }),
+                    ('windows:sid', ('it:os:windows:sid', {}), {
+                        'doc': 'The Microsoft Windows Security Identifier of the account.',
+                    }),
+                    ('groups', ('array', {'type': 'it:group'}), {
+                        'doc': 'An array of groups that the account is a member of.',
+                    }),
+                )),
+                ('it:group', {}, (
+                    ('name', ('str', {'lower': True, 'strip': True, 'onespace': True}), {
+                        'doc': 'The name of the group.',
+                    }),
+                    ('desc', ('str', {}), {
+                        'doc': 'A brief description of the group.',
+                    }),
+                    ('host', ('it:host', {}), {
+                        'doc': 'The host where the group is registered.',
+                    }),
+                    ('domain', ('it:domain', {}), {
+                        'doc': 'The authentication domain where the group is registered.',
+                    }),
+                    ('groups', ('array', {'type': 'it:group'}), {
+                        'doc': 'Groups that are a member of this group.',
+                    }),
+                    ('posix:gid', ('int', {}), {
+                        'doc': 'The primary group ID of the account.',
+                        'ex': '1001',
+                    }),
+                    ('windows:sid', ('it:os:windows:sid', {}), {
+                        'doc': 'The Microsoft Windows Security Identifier of the group.',
+                    }),
+                )),
+                ('it:logon', {}, (
+                    ('time', ('time', {}), {
+                        'doc': 'The time the logon occured.',
+                    }),
+                    ('success', ('bool', {}), {
```

Comment:
Since we don't use default values, it:logon:success is undefined unless explicitly set. What is the expected behavior for an undefined bool? Is the status/value of the property simply "unknown"?

Reply:
Yea, i think this is likely to only be filled in by orgs that are bothering to create `it:login` events that represent unsuccessful attempts.  So the idea would be that if you're filling it in, then annotate it on them all, and if you're not, then your analysis can assume they're all successful :)